q2cli/tools.py

```diff
@@ -80,6 +98,9 @@ def show_importable_types(ctx, param, value):
               callback=show_importable_types, expose_value=False,
               help='Show the semantic types that can be supplied to --type '
                    'to import data into an artifact.')
+@click.option('--show-importable-formats', is_flag=True, is_eager=True,
+              callback=show_importable_formats, expose_value=False,
+              help='Show all possible source formats available for import.')
```

Comment:
Gotcha, yeah that's better as it matches the precedent style from `importable types`.  

And done :D